cogs/mma.py

```diff
@@ -275,13 +277,21 @@ async def fight(self, ctx, *, optional_input: str=None):
         Use --schedule/sched                to fetch the next 5 events.
         Use --date YYYYMMDD                 to fetch events on a specific date.
         """
+
+        @staticmethod
+        async def fetch_json(url: str):
```

Comment:
Why not extract this method out of the `fight` method and into the class so that it can be used by other methods as well?